lib/flip_fab/version.rb

```diff
@@ -1,10 +1,7 @@
 module FlipFab
-  GEM_VERSION = '1.0.4'.freeze
+  VERSION = '1.0.5' # rubocop:disable Style/MutableConstant
 
-  # SB-specific versioning "algorithm" to accomodate Jenkins / gemstash
-  VERSION = if ENV['GEM_PRE_RELEASE'].nil? || ENV['GEM_PRE_RELEASE'].empty?
-                  FlipFab::GEM_VERSION
-                else
-                  "#{FlipFab::GEM_VERSION}.#{ENV['GEM_PRE_RELEASE']}"
-                end
+  # SB-specific versioning "algorithm" to accommodate BNW/Jenkins/gemstash
+  VERSION << '.' << ENV['GEM_PRE_RELEASE'].strip \
```

Comment:
Does changing a constant not create a warning? At least from rubocop?

Reply:
didn't see the rubocop disable.

Reply:
It's not really changing the `VERSION` constant, but rather the _(mutable)_ string object the constant is pointing at... nothing wrong with it from a Ruby point-of-view, but you're correct: Rubocop _(wrongly, IMO!)_ takes issue with it, which is why I added the `# rubocop:disable Style/MutableConstant` directive, to suppress the Rubocop warning.

Reply:
```ruby
[1] pry(main)> BLEGGA = 'foo'
=> "foo"
[2] pry(main)> BLEGGA << ' ' << 'bar'
=> "foo bar"
[3] pry(main)> BLEGGA
=> "foo bar"
[4] pry(main)> BLEGGA = 'qux'
(pry):4: warning: already initialized constant BLEGGA
(pry):1: warning: previous definition of BLEGGA was here
=> "qux"
[5] pry(main)> _
```